README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+# About Warren Casey Epstein
+## Undergraduate Student at the University of Chicago
+
+
+## Rising Third-Year in the College
+I am a rising third-year in the College, double majoring in **political science** and **economics** *with a specialization in business economics*. I will also hopefully minor in *near eastern languages and civilizations*.
+
+
+## Backround
+I am orginally from Fairfield, Connecitcut, but went to high school in Riverdale New York at SAR High School.
+
+
+## General Hobbies, interests, and likes/dislikes
+1. Eating all things pumpkin - fall is my favorite time of the year. **pumpkin ice cream**, **pumpkin spice lattes**, **pumpkin pie** the options are endless
+2. Sailing - when the never ending Chicago winter is over
+3. Hiking - also when the aforementioned Chicago winter is over
+
+## Some more favorites that are now affected by the Chicago Winter or season - pumpkin seems to go out sytle in the winter
+
+1. *To Kill a Mockingbird* by Harper Lee is my favorite book
+2. I enjoy reading *The Economist* and may have paid more money than I should have for a print copy of the magazine at Midway airport last week
+3. My favorite musical is *Come from Away* ![Image of Come from Away] https://images.app.goo.gl/EqriepACZkfi2mZs8
```

Comment:
Make sure that if you're linking an online photo, the page you've linked to is _intended_ to display that image. You linked to the Google Image Result here, so the photo itself won't render automatically.

Reply:
I also just noticed: in order to link to a page, the URL needs to be inside parentheses following the bracketed text (no space): 

 ```r
![Image of Come from Away](https://images.app.goo.gl/EqriepACZkfi2mZs8)
```

Renders as: 
![Image of Come from Away](https://images.app.goo.gl/EqriepACZkfi2mZs8)